### PR TITLE
Use UI types in the projection actor

### PR DESCRIPTION
**Pull request by klochowicz**

### CODE REVIEW

File: daemon/src/projection.rs [hunk 1]
```diff
@@ -87,7 +83,7 @@ struct State {
 }
 
 impl State {
-    pub fn to_cfd(&self) -> Vec<Cfd> {
+    pub fn as_cfds(&self) -> Vec<Cfd> {
```
Comment (thomaseizinger) [inline daemon/src/projection.rs]: Nit: `as_` usually means "cheap conversion of references". `to_cfds` was correct (reference to owned conversion).

See https://rust-lang.github.io/api-guidelines/naming.html#ad-hoc-conversions-follow-as_-to_-into_-conventions-c-conv.

**Review by thomaseizinger — APPROVED** [review]
Nice, thank you!

File: daemon/src/projection.rs (hunk 1 above)
Comment (klochowicz) [reply daemon/src/projection.rs]: thank you, I'll keep it in mind!

**Review by thomaseizinger — APPROVED** [review]

**Pull request merged**